source/slang/glsl.meta.slang

```diff
@@ -2787,47 +2787,1573 @@ public vec4 shadow2DProjLod(sampler2DShadow sampler, vec4 coord, float lod)
     return textureProjLod(sampler, coord, lod);
 }
 
-//
-// Ray tracing
-//
+
+//// RayTracing
+
+// Ray Tracing variables
 
 public typealias rayQueryEXT = RayQuery;
+public typealias accelerationStructureEXT = RaytracingAccelerationStructure;
+public typealias accelerationStructureNV = RaytracingAccelerationStructure;
+
+public typealias hitObjectNV = HitObject;
+
+//GL_EXT_ray_tracing BuiltIn's 
+
+void requireGLSLExtForRayTracingBuiltin() 
+{
+    __target_switch
+    {
+    case glsl:
+        __requireGLSLExtension("GL_EXT_ray_tracing");
+        __intrinsic_asm "";
+    }
+}
+
+__spirv_version(1.4) 
+void setupExtForRayTracingBuiltIn() 
+{
+    __target_switch
+    {
+    case glsl:
+        requireGLSLExtForRayTracingBuiltin();
+    case spirv:
+        return;
+    }
+}
+
+[require(glsl)]
```

Comment:
Please take a look at the capability definitions in `slang-capabilities.capdef` and properly mark these intrinsics with capaiblity requirements.

For example, `gl_LaunchIDNV` should be:
[require(spirv, raytracing)]
[require(glsl, raytracing)]